doc/en/announce/release-6.0.0.rst

```diff
@@ -0,0 +1,40 @@
+pytest-6.0.0
+=======================================
+
+The pytest team is proud to announce the 6.0.0 release!
+
+pytest is a mature Python testing tool with more than a 2000 tests
```

Comment:
Nit: no need for an indefinite article here:

```suggestion
pytest is a mature Python testing tool with more than 2000 tests
```

(See https://github.com/pytest-dev/pytest/pull/7552 to tidy the other files.)

Reply:
Indeed, thanks!